ui/css/itcss/components/send.scss

```diff
@@ -1,927 +0,0 @@
-@use "design-system";
-
-.send-screen-wrapper {
-  display: flex;
-  flex-flow: column nowrap;
-  z-index: 25;
-
-  @include design-system.screen-sm-max {
-    width: 100%;
-    overflow-y: auto;
-  }
-
-  section {
-    flex: 0 0 auto;
-  }
-}
-
-.send-screen-card {
-  background-color: var(--color-background-default);
-  box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
-  padding: 46px 40.5px 26px;
-  position: relative;
-  align-items: center;
-  display: flex;
-  flex-flow: column nowrap;
-  width: 498px;
-  flex: 1 0 auto;
-
-  @include design-system.screen-sm-max {
-    top: 0;
-    width: 100%;
-    box-shadow: none;
-    padding: 12px;
-  }
-}
-
-/* Send Screen */
-
-.send-screen section {
-  margin: 4px 16px;
-}
-
-.send-screen input {
-  @include design-system.H7;
-
-  width: 100%;
-}
-
-.send-eth-icon {
-  border-radius: 50%;
-  width: 70px;
-  height: 70px;
-  border: 1px solid var(--color-border-muted);
-  box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
-  position: absolute;
-  top: -35px;
-  z-index: 25;
-  padding: 4px;
-  background-color: var(--color-background-default);
-
-  @include design-system.screen-sm-max {
-    position: relative;
-    top: 0;
-  }
-}
-
-.send-screen-input-wrapper {
-  width: 95%;
-  position: relative;
-
-  .fa-bolt {
-    padding-right: 4px;
-  }
-
-  .send-screen-gas-input {
-    border: 1px solid transparent;
-  }
-
-  &__error-message {
-    display: none;
-  }
-
-  &--error {
-    input,
-    .send-screen-gas-input {
-      border-color: var(--color-error-default) !important;
-    }
-
-    .send-screen-input-wrapper__error-message {
-      @include design-system.H7;
-
-      display: block;
-      position: absolute;
-      bottom: 4px;
-      left: 8px;
-      color: var(--color-error-default);
-    }
-  }
-
-  .send-screen-input-wrapper__error-message {
-    @include design-system.H7;
-
-    display: block;
-    position: absolute;
-    bottom: 4px;
-    left: 8px;
-    color: var(--color-error-default);
-  }
```

Comment:
Not used

![Screenshot 2025-02-07 at 11 58 12 AM](https://github.com/user-attachments/assets/d89f3b3f-fcf6-4c6b-8867-79551c569d2a)